project.clj

```diff
@@ -100,6 +100,7 @@
 
   :profiles {:dev {:source-paths  ["dev"]
                    :dependencies  [[org.clojure/tools.namespace "0.2.5"]
+                                   [org.clojure/java.jmx "0.2.0"]
```

Comment:
@rlinehan Ah, yeah, that's the step I was missing.  I had added `org.clojure/java.jmx` as a top-level dep but still had it as a `:dev` profile dependency dep as well.  When I remove the reference to it from the `:dev` profile in tk-jetty9's project.clj but have it as a top-level dependency in tk-jetty9's project.clj, then it works like you mentioned.

Still haven't found a way to make it work as a "test" only dependency within tk-jetty9's project.clj, though.